core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -838,18 +838,6 @@ export class RealityModelTileClient {
       this.rdsProps.projectId = iTwinId;
   }
 
-  private async getAccessToken(): Promise<AccessToken | undefined> {
-    if (!IModelApp.authorizationClient)
-      return undefined;
-    let accessToken: AccessToken | undefined;
-    try {
-      accessToken = await IModelApp.authorizationClient.getAccessToken();

```

Comment:
@pmconne this is a similar problem. Maybe that's what you're seeing with reality data?

Reply:
Nope, still getting CORS errors.

Reply:
sad face emoji